tests/rest/media/test_media_retention.py

```diff
@@ -63,9 +63,9 @@ def _create_media_and_set_attributes(
             last_accessed_ms: Optional[int],
             is_quarantined: Optional[bool] = False,
             is_protected: Optional[bool] = False,
-        ) -> str:
+        ) -> MXCUri:
             # "Upload" some media to the local media store
-            mxc_uri = self.get_success(
+            mxc_uri: MXCUri = self.get_success(
```

Comment:
Aside: Is this annotation needed? I.e. Is get_success annotated enough for mypy to deduce this?)

(Fine as it is, I'm just curious)

Reply:
mypy does not actually complain about it - but PyCharm does fail to recognise `mxc_uri` without this annotation, oddly.

![image](https://user-images.githubusercontent.com/1342360/177576843-26f8c3af-73d8-4f32-8f1b-f670b82e5591.png)
![image](https://user-images.githubusercontent.com/1342360/177576880-116ae7eb-c063-4682-81e6-7091213a8e37.png)
![image](https://user-images.githubusercontent.com/1342360/177576899-22d90f3a-262a-4c1b-88bd-93f48ed11fb6.png)